application/ui/src/app/pages/matching-result/matching-result.page.ts

```diff
@@ -137,6 +141,63 @@ export class MatchingResultPage implements OnInit {
     this.matchingService.sendEmail(this.matchings).subscribe();
   }
 
+  makeSheduling () {
+    let matchSize;
+    let matchType;
+     
+    if ( this.matchingRequest !== undefined ) {
+      matchSize = this.matchingRequest.size;
+      matchType = "simple";
+    }
+    else{
+      matchSize = this.matchingGroupRequest.size;
+      matchType = "group";
+    }
+    console.log( "Bonjour" );
+    const sheduleParam: ScheduleParam = {
+      size: matchSize,
+      matchingType: matchType,
+      duration: this.duration+"",
+      time:this.time,
+      active: this.isrecurrence,
+      startDate: "2012-04-23T18:25:43.511Z",
+      endDate: "2012-04-23T18:25:43.511Z",
+      week: this.selectedWeek,
+      frequency: this.selectedPattern,
+      days: this.selectedEveryPattern
+    };
+    const schedulingRequest: SchedulingReq = {
+      organization: "dummy",
```

Comment:
you should not use dummy organization, compute the organization in the service to use it